app/controllers/admin/pages_controller.rb

```diff
@@ -0,0 +1,69 @@
+class Admin::PagesController < ApplicationController
+  before_action :require_admin
+  before_action :set_webbook, only: [:index, :show, :new, :edit, :update, :destroy]
```

Comment:
webbook があるのは前提なので、only は不要かなと。
```suggestion
  before_action :set_webbook
```